static/app/views/performance/newTraceDetails/utils.tsx

```diff
@@ -0,0 +1,51 @@
+import type {
```

Comment:
lets try and not introduce a generic utils file yet. The codebase is full of these and the code they contain can almost always be colocated someplace more relevant